docs/user-guide/commands.md

```diff
@@ -68,9 +68,18 @@ version: '1.0' # Major and Minor version number (patch is automatic), must be a
 description: |
   Lorem ipsum dolor sit amet.
 maintainer: foo@bar.com # Maintainer of the command
-format: binary # Format the command is in (binary)
+format: binary # Format the command is in (binary, habitat)
+# Case: binary
 binary:
     file: ./foobar.sh # Path to script or binary file from repository root
+# Case: habitat
+habitat:
```

Comment:
I think it's be more clear to me if we have separate examples

Binary example:
```
namespace: foo # Namespace for the command
name: bar # Command name
version: '1.0' # Major and Minor version number (patch is automatic), must be a string
description: |
  Lorem ipsum dolor sit amet.
maintainer: foo@bar.com # Maintainer of the command
format: binary # Format the command is in (binary, habitat)
binary:
    file: ./foobar.sh # Path to script or binary file from repository root
```
Remote Habitat example:
```
namespace: foo # Namespace for the command
name: bar # Command name
version: '1.0' # Major and Minor version number (patch is automatic), must be a string
description: |
  Lorem ipsum dolor sit amet.
maintainer: foo@bar.com # Maintainer of the command
format: habitat
habitat:
    package: core/node8 # Package of the Habitat command
    mode: remote # Mode the Habitat command is in (remote, local)
    command: node # Executable of the Habitat command
```
Local Habitat example: 
```
namespace: foo # Namespace for the command
name: bar # Command name
version: '1.0' # Major and Minor version number (patch is automatic), must be a string
description: |
  Lorem ipsum dolor sit amet.
maintainer: foo@bar.com # Maintainer of the command
format: habitat # Format the command is in (binary, habitat)
habitat:
    package: core/node8 # Package of the Habitat command
    mode: local # Mode of the Habitat command is in (remote, local)
    file: ./foobar.hart # Path to the .hart file from repository root
    command: node # Executable of the Habitat command
```

Reply:
It makes sense.  I separated examples.